cmd/antrea-agent/agent.go

```diff
@@ -744,10 +744,10 @@ func run(o *Options) error {
 		// Service would fail.
 		if o.config.AntreaProxy.ProxyAll {
 			klog.InfoS("Waiting for AntreaProxy to be ready")
-			if err := wait.PollUntil(time.Second, func() (bool, error) {
+			if err := wait.PollUntilContextCancel(ctx, time.Second, true, func(ctx context.Context) (bool, error) {
```

Comment:
Should this "immediate" be true for PollUtil?

Reply:
Yes, it should be set to `false`, I missed this one. Fixed.